<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>347</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>347</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Keyboard Input System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="bar_progress">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>260</y>
      <width>470</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_selectfile">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>80</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Map File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_selectboard">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>180</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Board to Alter Input</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_selectboard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>183</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_progstatus">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>240</y>
      <width>465</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_systemstatus">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>20</y>
      <width>149</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>System status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>50</y>
      <width>76</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopped</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_selectlayer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_selectlayer">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>100</y>
      <width>159</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Layer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fileloaded">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNew_Map_File"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionStart_Modification"/>
    <addaction name="actionStop_Modification"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <action name="actionNew_Map_File">
   <property name="text">
    <string>New Map File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionStart_Modification">
   <property name="text">
    <string>Start Modification</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop_Modification">
   <property name="text">
    <string>Stop Modification</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
